l1-contracts/scripts/token-migration.ts

```diff
@@ -218,3 +221,40 @@ async function prepareGovernanceTokenMigrationCall(
     executeCalldata,
   };
 }
+
+const provider = web3Provider();
+
+export async function transferTokensOnForkedNetwork(deployer: Deployer) {
+  const tokenList = ["0x5A520e593F89c908cd2bc27D928bc75913C55C42"];
+  for (const tokenAddress of tokenList) {
+    const erc20contract = IERC20Factory.connect(tokenAddress, provider);
+    console.log(`Migrating token ${tokenAddress}`);
+    console.log(
+      `Balance before: ${await erc20contract.balanceOf(deployer.addresses.Bridges.ERC20BridgeProxy)}, ${await erc20contract.balanceOf(deployer.addresses.Bridges.SharedBridgeProxy)}`
+    );
+    await transferTokens(deployer, tokenAddress);
+    console.log(
+      `Balance after: ${await erc20contract.balanceOf(deployer.addresses.Bridges.ERC20BridgeProxy)}, ${await erc20contract.balanceOf(deployer.addresses.Bridges.SharedBridgeProxy)}`
+    );
+  }
+  console.log("From 0", tokenList);
+  for (const tokenAddress of tokenList) {
+    const erc20contract = IERC20Factory.connect(tokenAddress, provider);
+    if (!(await erc20contract.balanceOf(deployer.addresses.Bridges.ERC20BridgeProxy)).eq(0)) {
+      console.log(`Failed to transfer all tokens ${tokenAddress}`);
+    }
+  }
+}
+
+export async function transferTokens(deployer: Deployer, token: string) {
```

Comment:
same here, explain why do we keep this function here and why its internal params such as "324" are hardcoded